react-apps/self-care-portal/self-care-portal-frontend/package.json

```diff
@@ -24,6 +24,7 @@
     "react": "^17.0.1",
     "react-axios": "^2.0.6",
     "react-bootstrap": "^1.4.0",
+    "react-bootstrap-icons": "^1.11.3",
```

Comment:
Where do we use these icons?

Reply:
This is used to import the `Download` icon in `profileMain.jsx`,

Reply:
Is it possible to use the download icon from the existing `fontawesome` library[1]?

[1] https://fontawesome.com/search?q=download&o=r